docs/serverless-v2/03-02-api-rules.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Exposing lambdas
+type: Details
+---
+
+By default, lambdas in Kyma are not exposed outside the cluster. All Function CRs receive the `serving.knative.dev/visibility=cluster-local` label that is also automatically assigned to the KSevice CR. Without this label, Knative Serving controller would create two virtual services, one for the internal cluster communication and the other one for exposing the lambda service outside the cluster. To restrict the access, we set the default `serving.knative.dev/visibility=cluster-local` label on KSevice CR that limits the Knative Serving controller to create only one, local virtual service that allows only for cluster-wide access to lambda services. Such a lambda is accessible for other resources within the cluster under the `{service-name}.{namespace}.svc.{domain}` endpoint, such as `test-lambda.default.svc.kyma.local`.
+
+To expose a lambda service outside the cluster, you need to specifically request it through an API Rule:
+
+![Expose a lambda service](./assets/api-rules.svg)
+
+1. Create an [APIRule custom resources (CR)](https://kyma-project.io/docs/components/api-gateway-v2/#custom-resource-api-rule) where you specify the lambda to expose, define a [Oathkeeper Access Rule](https://kyma-project.io/docs/components/api-gateway-v2/#details-available-security-options) to secure it, and list which HTTP request methods you want to enable for it.
```

Comment:
`...where you specify the k8s Service (in this case lambda) to expose...`

Reply:
As agreed, I will leave lambda here as this section refers specifically to them.